torch/csrc/jit/ir.cpp

```diff
@@ -1298,6 +1298,13 @@ Node* Graph::createDictIndex(Value* dict, Value* index) {
   return n;
 }
 
+Node* Graph::createStringIndex(Value* string, Value* index) {
```

Comment:
I don't think this is necessary, you can just the schematized insert, right? We should only add new `createX` methods for unschematized/special nodes.